enhancements/console/internationalization.md

```diff
@@ -0,0 +1,135 @@
+and translation of strings in the application. In addition, we have connected with the Red Hat
+globalization team for early-stage testing and to learn more about the translation process
+used for other products at Red Hat. We want to implement localization more widely in the console.
+
+## Motivation
+
+We seek to improve the user experience (UX) for a wider global range of customers.
+This change is important because it will allow us to meet the technical and cultural
+needs of multiple locales. For example, it will allow us to reach customers who would prefer using OpenShift in a different
+language (i.e. Chinese or Japanese) rather than English.
+
+Using the i18next library, we will have a single source code for all languages of the product and it will also be easier to
+leverage content strategists to author content and review copy consistency. All translatable strings in the front-end will be marked in the code and will be easy to export and modify.
+
+### Goals
+
+* Mark and extract to-be-translated strings in console
+* Work with the globalization team to obtain translations and perform localization testing for Chinese and Japanese
+* Work with UX to drive copy consistency in the UI (confirmation messages, actions that are more than one word, pieces from CoreOS, and similar actions with different verbiage)
+
+### Non-Goals
+
+We are not able to translate all text in the application. Text located in non-Red-Hat-controlled development environments may not be accessible for translation.
+This may include items such as events surfaced from Kuberenetes, alerts, and error messages displayed to the user or in logs.
+In addition, operators that surface informational messages may not have translations available, and localization of logging messages at any level is not in scope.
+Localization will not be included in the CLI at this time.
+
+## Proposal
+
+[i18next](https://www.i18next.com/) is a Javascript internationalization framework.
+It provides standard i18n features such as plurals, context, interpolation, and format,
+as well as language detection, loading, caching, and extensions.
+
+We have already [created a PR](https://github.com/openshift/console/pull/6194) integrating the i18next library into OpenShift.
+We have also been working with the Red Hat Globalization team to test out the hand-off process.
+
+We will need to mark all hard-coded strings with i18next so they can be extracted using an i18next extension and handed off to the Globalization team.
+This is not complex work but will involve many small code changes.
+[A rough string search](https://gist.github.com/jschuler/a404aed1ea774383cc2decd57d3135c6)
+of the frontend console code found about 9,000 strings, but many will not be translated (such as Kubernetes resource names).
+We will need to identify a prioritized list of UIs in OpenShift for localization and
+determine fallback behavior for pieces of the UI that are not translated.
```

Comment:
I think the fallback behavior is simply that the non-translated parts of the UI appear as English until we get to them.